clang/test/AST/ByteCode/strlen-unknown-size-array.cpp

```diff
@@ -0,0 +1,14 @@
+// Test that strlen() on external/unknown declarations doesn't crash the bytecode interpreter
+// This fixes issue #153948
+
+#include <cstddef>
+#include <cstdlib>
+#include <cstring>
```

Comment:
We can't use stdlib includes in the tests, you need to get the appropriate strcpy prototype and copy that in.